src/content/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic.mdx

```diff
@@ -13,50 +13,57 @@ redirects:
 
-This conceptual definition of "entity" is important because New Relic's goal is to give you practical information about the status of your business-important entities (and not just give you an unhelpfully large stream of assorted metrics and data). Our focus on entities, and the relationships between them, helps us optimize monitoring and troubleshooting of complex, modern systems.
+Our focus on entities and their relationships is important because our goal is to give you **practical information about your business-important entities**, and **not** give you an unhelpfully huge stream of data from a huge list of monitored things. With more insight at the entity level, you can better monitor and troubleshoot complex, modern systems.
 
-## Find and explore entities [#find]
+## Find and explore entities and entity data [#find]
 
-You'll find your entities wherever you see your data reporting in New Relic.
+Some tips for finding and understanding entity data:
 
-<Callout variant="tip">
-  You can create new entity types to monitor any data source. Learn more about [entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions).
-</Callout>
+* To find an entity's `entityGuid` and `entityName` and other metadata: from any list of monitored entities in the [New Relic Explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer-view-performance-across-apps-services-hosts), click an entity's <Icon name="fe-more-horizontal"/> icon, and click **See metadata and tags**.
+* For most entities, its GUID is reported as the attribute [`entityGuid`](/?event=SyntheticCheck&attribute=entityGuid). For workloads, it's `workloadGuid`. You can run [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language) to find entities by their GUID.
+* To see connections between entities, you have several options: 
+  * When viewing a specific entity in the UI, use the [**Related entities** UI](#related-entities). 
+  * Use [service maps](/docs/service-maps-dependencies-new-relic-one). 
+  * Use [distributed tracing](/docs/distributed-tracing-new-relic-one). 
+  * Use [our NerdGraph API to view entity relationships](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial).
+* To group entities together, see [Group entities](#group). 
+* [Configure entity definitions and relationships.](#entity-synthesis)
+* To learn technical details about entity types, see our [GitHub repo](https://github.com/newrelic/entity-definitions/tree/main/definitions). In an entity type's `definition` file, you'll see information like: 
+  * The `domain`: for example, `APM`, or `Infra`.
```

Comment:
Block letter after ":" .

Reply:
Sorry, I'm not sure what that one means

Reply:
We write the word in capitals after the :

Reply:
The `domain`: For example...

Reply:
oh i see. i was looking at wrong thing